### ISSUE TITLE
Normalization calculations in function detection_targets_graph

### ISSUE THREAD

**Issue by FruVirus** [matterport/Mask_RCNN#271]
In this function, there is an if block as follows:

if config.USE_MINI_MASK:
        y1, x1, y2, x2              = tf.split(positive_rois, 4, axis = 1)
        gt_y1, gt_x1, gt_y2, gt_x2  = tf.split(roi_gt_boxes, 4, axis = 1)
        gt_h                        = gt_y2 - gt_y1
        gt_w                        = gt_x2 - gt_x1
        y1                          = (y1 - gt_y1) / gt_h
        x1                          = (x1 - gt_x1) / gt_w
        y2                          = (y2 - gt_y1) / gt_h
        x2                          = (x2 - gt_x1) / gt_w
        boxes                       = tf.concat([y1, x1, y2, x2], 1)

What is the point of normalizing the ROI proposals this way? At this way, both y1 and gt_y1 are normalized to [0, 1] w.r.t. the input image coordinates. Why are we normalizing y1 again (and x1, y2, x2)?

**Comment by Superlee506** [comment]
@FruVirus I'm also confused about it. Do you know the reason now?

**Comment by FruVirus** [comment]
@Superlee506 , nope, still haven't figured this out....I've tried some detections with and without this block while config.USE_MINI_MASK is True and haven't found much difference

**Comment by FruVirus** [comment]
Also, we only do this if we use mini-mask since using mini-mask results in greater imprecise localization between jittered examples and the ground-truth whereas the imprecise localization from jittered examples using full-sized masks is tolerable.
